CHANGES.rst

```diff
@@ -39,6 +39,13 @@ tweakreg
 - Removed direct setting of the ``self.skip`` attribute from within the step
   itself. [#8600]
 
+- Updated requirement for ``tweakwcs`` to version ``0.8.8`` which fixes a crash
+  in the ``tweakreg`` step due to a ``MalformedPolygonError`` exception being
+  raised by the ``spherical_geometry`` package for some data sets. For more
+  details, see JP-3636 and
+  https://github.com/spacetelescope/tweakwcs/pull/205. [#8657]
```

Comment:
This is also user facing documentation and we don't keep references to JP. here. Can you replace the last sentence with 
`Fix is in spacetelescope/tweakwcs#205.`